services/src/main/resources/keycloak-default-client-profiles.json

```diff
@@ -110,6 +110,74 @@
           }
         }
       ]
+    },
+    {
+      "name" : "fapi-ciba-id1",
```

Comment:
Is it possible to use more generic name like "fapi-ciba" for this?

The reason is, that in the future once there is "FAPI CIBA Final" specification, we will be able to just modify this global client profile. But since it would have the `fapi-ciba-id1` in the name, we will possibly have some other things related to migration for this (Check all existing policies and rename the profile name in them in case they use it). Hence I would prefer more generic name not implying the `-id1`  suffix

Reply:
@mposolda I agree with you. I'll rename it to `fapi-ciba`.